numpy/_core/tests/test_dlpack.py

```diff
@@ -21,6 +33,13 @@ def test_dunder_dlpack_stream(self):
         with pytest.raises(RuntimeError):
             x.__dlpack__(stream=1)
 
+    def test_dunder_dlpack_copy(self):
+        x = np.arange(5)
+        x.__dlpack__(stream=None)
+
+        with pytest.raises(RuntimeError):
+            x.__dlpack__(stream=1)
+
```

Comment:
Did you mean to test `copy` here?

Reply:
Fixed, `copy=` parsing is too forgiving, IMO, but I opened an issue on that.  It doesn't matter here.